tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2030,7 +2027,7 @@ p {
   }
   @if (($type) == (ocean)) {
   }
-  @if (($type)==(ocean)) {
+  @if (($type) ==(ocean)) {
```

Comment:
Problem in parser, he is think this is function, maybe need fix in parser, but we use loose flag and in loose this maybe interpreted as function. What do you think?